jax/_src/numpy/lax_numpy.py

```diff
@@ -1832,10 +1832,44 @@ def _moveaxis(a: Array, source: tuple[int, ...], destination: tuple[int, ...]) -
   return lax.transpose(a, perm)
 
 
-@util.implements(np.isclose)
 @partial(jit, static_argnames=('equal_nan',))
 def isclose(a: ArrayLike, b: ArrayLike, rtol: ArrayLike = 1e-05, atol: ArrayLike = 1e-08,
             equal_nan: bool = False) -> Array:
+  """Check if two arrays are element-wise approximately equal within a tolerance.
+
+  JAX implementation of :func:`numpy.allclose`.
+
+  Infs in ``a`` will be considered equal to Infs in ``b``.
+
+  Args:
+    a: first input array to compare.
+    b: second input array to compare.
+    rtol: relative tolerance used for approximate equality. Default = 1e-05.
+    atol: absolute tolerance used for approximate equality. Default = 1e-08.
```

Comment:
We should explain what relative and absolute tolerance mean; essentially this function evaluates the following condition:

$|a - b| \le \mathtt{atol} + \mathtt{rtol} * |b|$